src/main/java/Task5.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.StringJoiner;
+import java.util.ArrayList;
+
+public class Task5 {
```

Comment:
Please apply "Reformat code"  action from IntelliJ Idea (Ctrl + Alt + L shortcut or from Code -> Reformat Code context menu) to the whole class

Reply:
Commit with Code reformatting to Task5  and TestTask5 Classes are added